style.css

```diff
@@ -1,588 +1,983 @@
+  justify-content: center;
+  padding: 24px;
+  width: 100%;
 }
 
-.contact {
-  background-color: #fff;
-  padding: 0 24px;
+.footer-line {
+  width: 100%;
+  height: 1px;
+  margin-bottom: 25px;
+}
+
+@media (min-width: 768px) {
+  .menu {
+    display: none;
+  }
+
+  .humb {
+    display: none;
+  }
+
+  .menu-desktop-container {
+    display: flex;
+    flex-direction: row;
+    margin-right: 100px;
+  }
+
+  .menu-desktop {
+    font-family: 'Inter', sans-serif;
+    font-size: 15px;
+    text-align: center;
+    color: #fff;
+    text-decoration: none;
+    margin-left: 32px;
+  }
+
+  .logo {
+    margin-left: 100px;
+  }
+
+  .page1-content {
+    display: grid;
+    grid-template-columns: 5% 45% 50%;
+    height: fit-content;
+    padding-bottom: 20%;
+    width: 100%;
+    padding-top: 3%;
+  }
+
+  .header-text {
+    grid-column-start: 2;
+    margin-left: 4%;
+  }
+
+  .my-name {
+    display: flex;
+    align-self: center;
+    font-size: 62px;
+    margin-bottom: 13%;
+  }
+
+  .my-position {
+    font-size: 32;
+    margin-bottom: 30px;
+  }
+
+  .discription-me {
+    font-size: 20px;
+    margin-bottom: 30px;
+  }
+
+  /* Add animation below */
+  .animate-btn {
+    -webkit-animation-duration: 0.8s;
+    animation-duration: 0.8s;
+    transition: background 0.8s ease-in-out, color 0.8s ease-in-out;
+  }
+
+  .animate-btn:focus {
+    background: #e05304;
+    color: #f4f1f0;
+  }
+
+  .animate-btn:hover {
+    background: #fff5f1;
+    border: 1px solid #f55800;
+    color: #f55800;
+  }
+
+  @-webkit-keyframes animate-btn-hover {
+    100% {
+      box-shadow: inset 11rem 0 0 0 #6070ff, 0 8px 16px rgba(64, 83, 252, 0.24);
+      color: #fff;
+    }
+  }
+
+  @keyframes animate-btn-hover {
+    100% {
+      box-shadow: inset 11rem 0 0 0 #6070ff, 0 8px 16px rgba(64, 83, 252, 0.24);
+      color: #fff;
+    }
+  }
+
+  .animate-logo {
+    color: #6070ff;
+    animation-name: rotate-logo;
+    animation-duration: 1.5s;
+    animation-iteration-count: 4;
+  }
+
+  @keyframes rotate-logo {
+    50% {
+      transform: scale(1.5) rotate(360deg) translateX(-50%);
+    }
+
+    to {
+      transform: scale(1) rotate(720deg) translateX(0);
+    }
+  }
+
+  .img-list {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .desktop-b-img-1 {
+    display: flex;
+    grid-column-start: 2;
+    grid-column-end: 3;
+    position: absolute;
+    bottom: 12%;
+    right: 0;
+    width: 50%;
+    height: auto;
+    top: 70px;
+  }
+
+  .b-img-1 {
+    display: none;
+  }
+
+  .page-footer {
+    position: absolute;
+    left: 24px;
+    display: flex;
+    align-self: center;
+    width: 5%;
+  }
+
+  .projects-subject {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    width: 100%;
+    padding: 100px;
+  }
+
+  .projects {
+    display: flex;
+    margin-top: 0;
+  }
+
+  .line {
+    width: 40%;
+    margin-left: 5px;
+  }
+
+  .project-1 {
+    display: grid;
+    grid-template-columns: 60% 40%;
+  }
```

Comment:
This is the way this project is looking in your design.

<img width="1440" alt="Screen Shot 2022-06-11 at 9 04 11" src="https://user-images.githubusercontent.com/82426682/173191167-8eb11613-b617-4a10-8b8e-18be5b7dd216.png">

Kindly fix it.